packages/wrangler/src/index.ts

```diff
@@ -374,6 +375,9 @@ export function createCLIParser(argv: string[]) {
 		routeHandler
 	);
 
+	// wrangler.command(CloudchamberCommand);
+	wrangler.command("cloudchamber", false, CloudchamberCommand);
```

Comment:
This leads to a suboptimal experience when running `wrangler cloudchamber` (no args). For most commands (e.g. `wrangler r2`) not specifying a subcommand will lead to help output e.g.

```
wrangler r2

📦 Interact with an R2 store

Commands:
  wrangler r2 object  Manage R2 objects
  wrangler r2 bucket  Manage R2 buckets
```

For `wrangler cloudchamber` it leads to help output but _also_ includes a non-actionable error:

```
✘ [ERROR] Not enough non-option arguments: got 0, need at least 1


wrangler cloudchamber
```

Could you integrate with `subHelp`?